modules/fundamental/src/product/model/mod.rs

```diff
@@ -54,13 +54,13 @@ impl ProductVersionHead {
     }
 
     pub async fn from_entities(
-        product_versions: Vec<product_version::Model>,
+        product_versions: &Vec<product_version::Model>,
```

Comment:
I find clippy telling me to use `&[product_version::Model]` which you can get by `&`'ing a `Vec`.